staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -214,14 +212,30 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 	s.GenericAPIServer.Handler.NonGoRestfulMux.UnlistedHandle("/apis/", apisHandler)
 
 	apiserviceRegistrationController := NewAPIServiceRegistrationController(informerFactory.Apiregistration().V1().APIServices(), s)
+	if len(c.ExtraConfig.ProxyClientCertFile) > 0 && len(c.ExtraConfig.ProxyClientKeyFile) > 0 {
```

Comment:
we need to fail if the files aren't specified. That would match previous behavior, right?

Reply:
I identified only https://github.com/kubernetes/kubernetes/pull/92791/files#diff-c070121c775ca8660b40b691e24bc6c4L156 as a potential place that could have failed when the files were missing.